scripts/myst_cc

```diff
@@ -12,16 +12,30 @@
 MYST_ROOT="$(realpath "$(dirname "$0")/..")"
 
 cp -r "${MYST_ROOT}"/build/obj/tools/* -t "${MYST_ROOT}"/tools
+cp -r "${MYST_ROOT}"/build/obj/utils/* -t "${MYST_ROOT}"/utils
+cp -r "${MYST_ROOT}"/build/obj/hostfs/* -t "${MYST_ROOT}"/hostfs
+cp -r "${MYST_ROOT}"/build/obj/host/* -t "${MYST_ROOT}"/host
+cp -r "${MYST_ROOT}"/build/obj/ext2/* -t "${MYST_ROOT}"/ext2
 cp -r "${MYST_ROOT}"/build/obj/kernel/* -t "${MYST_ROOT}"/kernel
-cp -r "${MYST_ROOT}"/build/obj/crt/"${MYST_ROOT}"/third_party/* -t "${MYST_ROOT}"/third_party
+cp -r "${MYST_ROOT}"/build/obj/oe/* -t "${MYST_ROOT}"/oe
+cp -r "${MYST_ROOT}"/build/obj/json/* -t "${MYST_ROOT}"/json
+cp -r "${MYST_ROOT}"/build/obj/target/* -t "${MYST_ROOT}"/target
+# cp -r "${MYST_ROOT}"/build/obj/crt/"${MYST_ROOT}"/third_party/* -t "${MYST_ROOT}"/third_party
```

Comment:
Do we want code coverage for MUSL?

Reply:
Not at the moment, we discussed putting in a version that would identify the files that we change using the patch and produce coverage results only for those files.